README.md

```diff
@@ -2,16 +2,18 @@
 
 ## Ruby's TIC TAC TOE
 
-> This is a Tic Tac Toe game all made with ruby as a project, the is made to work for 2 players, each player needs to choose on which location they want to add its mark until one of them has 3 marks on a row, and he will be declared the winner.
+<img src="https://github.com/JAAR91/TIC-TAC-TOE/blob/feature/readme_game_instructions/lib/game_example.gif">
+
 
+> This is a Tic Tac Toe game all made with ruby as a project, the is made to work for 2 players, each player needs to choose on which location they want to add its mark until one of them has 3 marks on a row, and he will be declared the winner.
 
 ## Built With
 
-- Ruby
+- >Ruby
 
 ## Getting Started
 
-To get a local copy up and running follow these simple example steps.
+> To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] You can put the instructions to clone the repository here, right now there is no logic order